pages/download.py

```diff
@@ -87,13 +87,26 @@ def __init__(self):
         self.title = _("Download")
         self.slug = "download"
     def get_content(self):
+        download_intro = md(_("""
+To access Freenet, you first need to install the main application. Freenet will run in the background and you can use
+your browser to change settings and access content. There are other applications that you can install
+at a later time to add more functionality.
+
+Steps:
+
+* [Download and install Freenet](#autostart)
+* [Add friends (or connect to strangers)](#note)
+* [Start using Freenet!](#usage)
```

Comment:
URLs may change in the future and should therefore not be hardcoded in the translation unit. Please use [reference-style link syntax](http://daringfireball.net/projects/markdown/syntax#link) `[link text][link_identifier]` and append an index paragraph with the URL identifiers.